src/components/Carousel.js

```diff
@@ -430,9 +430,11 @@ class Carousel extends Component {
         return images && images[selectedItem];
     }
 
-    getVariableImageHeight = (position) => {
-        const item = this.refs[`item${position}`];
-        const images = item && item.getElementsByTagName('img');
+    getVariableHeight = (position) => {
+				const item = this.refs[`item${position}`];
+				const images = item && item.getElementsByTagName('img');
+				const divs = item && item.getElementsByTagName('div');
```

Comment:
There is no requirement saying that each slide should be a div. We just iterate over children and then build a list of slides. This change wouldn't be robust enough.

Reply:
Would just using clientHeight of the <li> work? I guess might need to adjust CSS to ensure no padding but would account for entire slide no matter the content as long as nothing out-of-flow

Reply:
Sounds like a good idea. Need to be tested, though.